src/components/taskDetails/ProgressUpdateCard/LatestProgressUpdateCardPresentation.tsx

```diff
@@ -65,7 +71,14 @@ export default function LatestProgressUpdateCardPresentation({
                 >
                     <Tooltip
                         content={tooltipText}
-                        tooltipPosition={{ top: '-2.6rem', right: '-4rem' }}
+                        tooltipPosition={
+                            isDevMode
+                                ? { bottom: '2rem', right: '-2rem' }
+                                : {
+                                      bottom: '1.4rem',
+                                      right: '-5rem',
+                                  }
+                        }
```

Comment:
why are you changing the tooltip position ?

Reply:
I am adding the user avatar right to the date component so i have to change the tooltip position to the left and also the tooltip positing was not very accurate previously so i have just make some slight changes to make it more accurate.